article.html

```diff
@@ -160,7 +160,7 @@
 		<a href="" class="comments-count">12</a>
 		<h4>Add a Comment</h4>
 		<p><textarea rows="5"></textarea></p>
-		<button>ADD</button> <a href="" class="anon">ADD anonymously</a>
+		<button>ADD</button> <button class="anon">ADD anonymously</button>
```

Comment:
Can you style these 2 as `<input type='submit' value='ADD' /> <input type='submit' value='ADD anonymously' />`